recipes/ribocode/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: ribocode
+  version: 1.2.6
+
+source:
+  fn: RiboCode-1.2.6.tar.gz
+  url: https://pypi.python.org/packages/c6/4b/902412b7c9683bcb9096e9359d17c6c0538b084a0c3167603ed69278aeb6/RiboCode-1.2.6.tar.gz
+  md5: e05ef843da351c4588e02bad3d5c3d65
+
+requirements:
+  build:
+    - python # [py27]
```

Comment:
You should include something like that:
```
build:
  skip: true  # [not py27]
```

Please have a look at other packages or our guidelines.